vismne/viz.py

```diff
@@ -1,36 +1,77 @@
+               'medial': {'v': (90., 0.), 'r': -90., 'xyz': [1, 2]},
+               'rostral': {'v': (180., 0.), 'r': -180., 'xyz': [0, 2]},
+               'caudal': {'v': (0., 0.), 'r': 0., 'xyz': [0, 2]},
+               'dorsal': {'v': (0., 90.), 'r': 90., 'xyz': [0, 1]},
+               'ventral': {'v': (0., -90.), 'r': 90., 'xyz': [0, 1]},
+               'frontal': {'v': (120., 80.), 'r': 106.739, 'xyz': [0, 1]},
+               'parietal': {'v': (-120., 60.), 'r': 49.106, 'xyz': [0, 1]}}
+rh_viewdict = {'lateral': {'v': (90., 0.), 'r': -90., 'xyz': [1, 2]},
+               'medial': {'v': (-90., 0.), 'r': 90., 'xyz': [1, 2]},
+               'rostral': {'v': (180., 0.), 'r': 180., 'xyz': [0, 2]},
+               'caudal': {'v': (0., 0.), 'r': 0., 'xyz': [0, 2]},
+               'dorsal': {'v': (0., 90.), 'r': 90., 'xyz': [0, 1]},
+               'ventral': {'v': (0., -90.), 'r': 90., 'xyz': [0, 1]},
+               'frontal': {'v': (60., 80.), 'r': -106.739, 'xyz': [0, 1]},
+               'parietal': {'v': (-60., 60.), 'r': -49.106, 'xyz': [0, 1]}}
 viewdicts = dict(lh=lh_viewdict, rh=rh_viewdict)
 
 
 class Brain(object):
-    """docstring for Brain."""
-
-    def __init__(self, subject_id, hemi, surf, title=None, views=['lat'],
-                 offset=True):
+    """Class for visualizing a brain using multiple views in vispy.
+
+    Parameters
+    ----------
+    subject_id : str
+        subject name in Freesurfer subjects dir
+    hemi : str
+        hemisphere id (ie 'lh', 'rh', 'both', or 'split'). In the case
+        of 'both', both hemispheres are shown in the same window.
+        In the case of 'split' hemispheres are displayed side-by-side
+        in different viewing panes.
+    surf : str
+        freesurfer surface mesh name (ie 'white', 'inflated', etc.)
+    title : str
+        title for the window
+    views : list | str
+        views to use (lateral, medial, rostral, caudal, dorsal, ventral,
+        frontal parietal)
+    offset : bool
+        If True, aligs origin with medial wall. Useful for viewing inflated
+        surface where hemispheres typically overlap (Default: True)
+    subjects_dir : str | None
+        If not None, this directory will be used as the subjects directory
+        instead of the value set using the SUBJECTS_DIR environment variable.
+    units : str
+        Can be 'm' or 'mm' (default).
+    size : float or pair of floats
+        the size of the window, in pixels. can be one number to specify
+        a square window, or the (width, height) of a rectangular window.
+    background : matplotlib color
+        Color of the background.
+    foreground : matplotlib color
+        Color of the foreground (will be used for colorbars and text).
+        None (default) will use black or white depending on the value
+        of ``background``.
+    """
+
+    def __init__(self, subject_id, hemi, surf, title=None, views=['lateral'],
```

Comment:
using a list as a default value is not good practice. See here: https://docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html

Reply:
also you should do some input validation. That is, if the user gives `views='lat'`, it should create a list internally ...